django/core/management/commands/test.py

```diff
@@ -80,6 +80,10 @@ def handle(self, *test_labels, **options):
         from django.conf import settings
         from django.test.utils import get_runner
 
+        # Don't run migrations for tests
+        if settings.DATABASES.get('migration') is not None:
```

Comment:
I'm not aware of Django setting a key called 'migration' in `settings.DATABASES`?